src/hooks/tests/use-import-export.test.tsx

```diff
@@ -36,6 +36,7 @@ beforeEach( () => {
 	} );
 	( useSiteDetails as jest.Mock ).mockReturnValue( {
 		updateSite: jest.fn(),
+		startServer: jest.fn(),
```

Comment:
What do you think if we add a new test or at least a new assert to check we called the startServer after an import?

Something like: `expect( useSiteDetails().startServer ).toHaveBeenCalledTimes( 1 );`

Reply:
Thanks for testing and reviewing the PR, Antonio!

> What do you think if we add a new test or at least a new assert to check we called the startServer after an import?
> Something like: expect( useSiteDetails().startServer ).toHaveBeenCalledTimes( 1 );

I like the idea! I have made a change in https://github.com/Automattic/studio/pull/678/commits/cd93a5f45daa766f27102d2bba005bf036522ee9 and also added testing whether `startServer` was called with the correct site ID.